contrib/scripts/kind.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+# Sample usages
+# ./contrib/scripts/kind.sh
+# IPv6=1 ./contrib/scripts/kind.sh
+# NO_BUILD=1 ./contrib/scripts/kind.sh
+# NO_PROVISION=1 ./contrib/scripts/kind.sh
+
+# Enable IPv6 mode. By default using IPv4
+export 'IPv6'="${IPv6:0}"
+# Set NO_BUILD=1 to avoid building docker images again
+export 'NO_BUILD'="${NO_BUILD:0}"
+# Set NO_PROVISION=1 to use existing kind cluster
+export 'NO_PROVISION'="${NO_PROVISION:0}"
+
+# Build cilium and cilium operator docker images
+if [[ "${NO_BUILD}" -ne "1" ]]; then
+  make -j docker-cilium-image docker-operator-generic-image
+fi
+
+# Provision kind cluster if required
+if [[ "${NO_PROVISION}" -ne "1" ]]; then
+  kind delete cluster
+  # clean up previous kind network
+  docker system prune -f
+
+  if [[ "${IPV6}" -ne "1" ]]; then
+    kind_config_file=.github/kind-config.yaml
+  else
+    kind_config_file=.github/kind-config-ipv6.yaml
+  fi
+
+  kind create cluster --config=$kind_config_file
```

Comment:
```suggestion
  kind create cluster --config="$kind_config_file"
```

Not strictly necessary here, but in case the filename changes in the future?